### PR TITLE
[11.0][MIG] purchase_location_by_line: Migration to 11.0

### PR DESCRIPTION
**Pull request by hizbul25**
This module is depend on `purchase_delivery_split_date`. I have changed that module and change some text and version this module. Please accept pull request.

Depends on:

-  [ ] purchase_delivery_split_date - #511

### CODE REVIEW

**Review by gurneyalex — DISMISSED** [review]

File: purchase_location_by_line/__init__.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
```
Comment (nikul-serpentcs) [inline purchase_location_by_line/__init__.py:1]: @rvalyi  IMO remove Copyright in all __init__.py file.

File: purchase_location_by_line/views/purchase_view.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            </xpath>
+        </field>
+    </record>
+
+    <record id="purchase_order_line_form2" model="ir.ui.view">
+        <field name="name">purchase.order.line.form</field>
+        <field name="model">purchase.order.line</field>
+        <field name="inherit_id" ref="purchase.purchase_order_line_form2"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='date_planned']" position="after">
```
Comment (nikul-serpentcs) [inline purchase_location_by_line/views/purchase_view.xml:20]: @hizbul25 IMO Directly use `field`

File: purchase_location_by_line/views/purchase_view.xml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            </xpath>
+        </field>
+    </record>    
+
+    <record id="purchase_order_line_tree" model="ir.ui.view">
+        <field name="name">purchase.order.line.tree</field>
+        <field name="model">purchase.order.line</field>
+        <field name="inherit_id" ref="purchase.purchase_order_line_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='date_planned']" position="after">
```
Comment (nikul-serpentcs) [inline purchase_location_by_line/views/purchase_view.xml:31]: @hizbul25 same here

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Pull request closed without merging**